tests/Microsoft.Graph.DotnetCore.Core.Test/Mocks/MockUserEventsCollectionPage.cs

```diff
@@ -0,0 +1,31 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using Microsoft.Graph.DotnetCore.Core.Test.TestModels.ServiceModels;
+
+namespace Microsoft.Graph.DotnetCore.Test.Mocks
+{
+    using Microsoft.Graph.DotnetCore.Test.Tasks;
+    using System;
+    using System.Collections.Generic;
+    public class MockUserEventsCollectionPage : CollectionPage<TestEvent>, ITestEventDeltaCollectionPage
+    {
+        public MockUserEventsCollectionPage(IList<TestEvent> currentPage, TestEventDeltaRequest nextPageRequest, string linkType = "") : base(currentPage)
```

Comment:
could we replace this class by a mocking framework?

Reply:
Will investigate this.